### PR TITLE
Added management command to update course images

### PR DESCRIPTION
**Pull request by clintonb**
This command downloads images from the course's card_image_url field
value, and stores them in the image field.

LEARNER-2463

### CODE REVIEW

File: course_discovery/apps/course_metadata/management/commands/download_course_images.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        courses = Course.objects.filter(card_image_url__isnull=False).exclude(card_image_url='').order_by('key')
+
+        if not options['overwrite_existing']:
+            courses = courses.filter(image='')
+
+        count = courses.count()
+        if count:
+            logger.info('Retrieving images for [%d] courses...', count)
+        else:
+            logger.info('All courses are up to date.')
```
Comment (schenedx) [inline course_discovery/apps/course_metadata/management/commands/download_course_images.py]: `return` after this statement?

**Review by schenedx — COMMENTED** [review]
Why creating a new management command, but not having this as part of `refresh_course_metadata` management command?

**Comment by clintonb** [comment]
I opted for a new command so that we could perform this task optionally. Realistically, we only need to run it once.

**Review by schenedx — APPROVED** [review]
👍

**Pull request merged**